lib/pronto/formatter/formatter.rb

```diff
@@ -1,30 +1,32 @@
 module Pronto
   module Formatter
-    def self.get(names)
-      names ||= 'text'
-      Array(names).map { |name| FORMATTERS[name.to_s] || TextFormatter }
-        .uniq.map(&:new)
-    end
+    class << self
+      def add(label, formatter_klass = nil)
```

Comment:
```suggestion
      def register(label, formatter_klass)
```
**suggestion**: why do you want to make `formatter_klass` optional? It would raise an exception below anyway :man_shrugging:

Also, `register` sounds a much better name as we're not providing `remove` method.

Reply:
Good catch, thanks!
I forgot to remove `= nil`.
Agreed, `register` is a better name.